extensions/dacpac/src/wizard/pages/deployConfigPage.ts

```diff
@@ -135,8 +136,11 @@ export class DeployConfigPage extends DacFxConfigPage {
 			this.model.database = this.databaseTextBox.value;
 			this.instance.setDoneButton(Operation.deploy);
 
-			// remove deploy plan page
+			// remove deploy plan page and readd summary page so that it has the correct page number
+			this.instance.wizard.removePage(DeployOperationPath.summary);
```

Comment:
noticed that in upgradeRadioButton.onDidClick we are adding only the deployplan page... do we add only the very next page? Iso... may be we should check if summary page is added or not before removing?

Reply:
ah good catch! That summary page also has to be removed and re-added there to have the correct page number